store/tikv/client.go

```diff
@@ -45,105 +55,230 @@ const (
 	readTimeoutShort  = 20 * time.Second  // For requests that read/write several key-values.
 	readTimeoutMedium = 60 * time.Second  // For requests that may need scan region.
 	readTimeoutLong   = 150 * time.Second // For requests that may need scan region multiple times.
+
+	rpcLabelKV  = "kv"
+	rpcLabelCop = "cop"
 )
 
 type rpcClient struct {
-	msgID uint64
-	p     *Pools
+	p *Pools
```

Comment:
I think we can discard the pool now, maintaining 1 conn with each tikv-server should be enough.